css/main.css

```diff
@@ -1,3 +1,139 @@
 /* Add your custom CSS here */
 
 /* Dont' forget to link this file to your HTML in the <head> */
+h1, h2, h3 {
+font-family: 'Cinzel', serif;
+}
+
+body {
+    font-family: 'Quicksand', sans-serif;
+}
+
+ul {
+    list-style: none;
+}
+
+/* .li {
+display: flex;
+} */
+
+
+.hero-image {
+    background-image: url(https://images.unsplash.com/photo-1514533450685-4493e01d1fdc?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1934&q=80);
+
+    background-color: #cccccc; /* Used if the image is unavailable */
+    height: 500px; /* You must set a specified height */
+    background-position: center; /* Center the image */
+    background-repeat: no-repeat; /* Do not repeat the image */
+    background-size: cover;
+    padding-top: 90px;
+    text-align: left;
+    
+}
+
+.ticket-button {
+    color: white;
+    background-color: rgb(84, 72, 187);
+    margin-top:20px;
+    font-size: 16px;
+    border: none;
+    padding: 15px 30px;
+
+}
+
+h1, h2, h3, p {
+color: white;
+}
+
+.navy-blue {
+    color: #190F3F;
+    /* padding-top:5px; */
+    padding-bottom:10px;
+}
+
+.line-up-section {
+    background-image: url(../img/festival_lilly-flower.svg), url(../img/festival_blob-shape.svg);
+    background-repeat: no-repeat, no-repeat;
+    background-position: left bottom, center;
+    background-size: 90px, contain;
+    background-color: rgb(97, 153, 32);
```

Comment:
A little creative licence is ok as this is just an exercise, but bear in mind that the task (and the job of a front-end developer!) is to re-create designs exactly, included colours...